pkg/function/examples/even_odd/main.go

```diff
@@ -16,9 +16,9 @@ func handle(_ context.Context, keys []string, d functionsdk.Datum) functionsdk.M
 	if num, err := strconv.Atoi(string(msg)); err != nil {
 		return functionsdk.MessagesBuilder().Append(functionsdk.MessageToDrop())
 	} else if num%2 == 0 {
-		return functionsdk.MessagesBuilder().Append(functionsdk.MessageTo([]string{"even"}, msg))
+		return functionsdk.MessagesBuilder().Append(functionsdk.NewMessage(msg).WithKeys([]string{"even"}))
```

Comment:
Is this example used for conditional forwarding? Does it need tags as well?

Reply:
yeah, I will add tags